tests/lustre/typechecking/shouldPass/test_tuples.lus

```diff
@@ -0,0 +1,19 @@
+node n1 (x: [int, real]) returns (y : [bool, int]);
+
+node g (x:int) returns (y:int);
```

Comment:
Please convert this file in one or more correct Lustre programs.

> line 3 col. 4: Failure cause: Syntax Error!

Reply:
This was our tuples discussion. I will take this file out of tracking :)